### PR TITLE
Added task-dependency data model

### LINKED ISSUE #45: Add Data Model for TaskDependencies collection

### PR DESCRIPTION
**Pull request by vinit717**
close #45
Task-dependency model is used to store dependencies of task (like which task is depends on which task)

### CODE REVIEW

**Review by sahsisunny — APPROVED** [review]

File: task-dependency/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Firestore taskDependencies collection data model
+
+```json
+{
+  "taskId" : "Id of task which has dependencies",
+  "dependsId" : "Id of task which a task is dependent On"
+}
```
Comment (heyrandhir) [inline task-dependency/README.md:7]:
```suggestion
{
  "taskId" : "Id of the task which has dependencies.",
  "dependsId" : "Id of the task which the task is dependent on."
}

**Review by heyrandhir — CHANGES_REQUESTED** [review]

File: task-dependency/README.md (hunk 1 above)
Comment (vinit717) [reply task-dependency/README.md]: done

**Review by heyrandhir — DISMISSED** [review]

File: task-dependency/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Firestore taskDependencies collection data model
+
+```json
+{
+  "taskId" : "Id of the task which has dependencies.",
+  "dependsId" : "Id of the task which the task is dependent on."
```
Comment (RitikJaiswal75) [inline task-dependency/README.md:6]: please change the name of the key

**Review by heyrandhir — COMMENTED** [review]
Description please

File: task-dependency/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+}
+```
+
+## Example 
+
+```json
+{
+  "dependsId" : "bIkwsOXXWNOjU2bGQ64Y",
```
Comment (RitikJaiswal75) [inline task-dependency/README.md:14]: please change this key

**Review by RitikJaiswal75 — CHANGES_REQUESTED** [review]

File: task-dependency/README.md (hunk 3 above)
Comment (vinit717) [reply task-dependency/README.md]: done

File: task-dependency/README.md (hunk 2 above)
Comment (vinit717) [reply task-dependency/README.md]: done

**Review by RitikJaiswal75 — APPROVED** [review]

**Review by sahsisunny — APPROVED** [review]

**Review by heyrandhir — APPROVED** [review]

**Pull request merged**